deploy/charts/cert-manager/webhook/templates/service.yaml

```diff
@@ -4,16 +4,17 @@ metadata:
   name: {{ include "webhook.fullname" . }}
   namespace: {{ .Release.Namespace | quote }}
   labels:
-    app: {{ include "webhook.name" . }}
-    chart: {{ include "webhook.chart" . }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
+    app.kubernetes.io/name: {{ include "webhook.name" . }}
+    app.kubernetes.io/instance:  {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ include "webhook.chart" . }}
 spec:
   type: ClusterIP
   ports:
   - name: https
     port: 443
     targetPort: 6443
   selector:
-    app: {{ include "webhook.name" . }}
-    release: {{ .Release.Name }}
+    app.kubernetes.io/name: {{ include "webhook.name" . }}
+    app.kubernetes.io/instance:  {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
```

Comment:
Changing selectors here should be fine, although it *will* cause a brief period of outage for the webhook. I don't think that'll cause any issues for users though 😄

Reply:
the time to replace the pod :)